src/main/java/org/opentripplanner/routing/algorithm/mapping/AlertToLegMapper.java

```diff
@@ -40,6 +40,18 @@ public static void addAlertPatchesToLeg(Graph graph, Leg leg, boolean isFirstLeg
                 addAlertPatchesToLeg(leg, StopCondition.ARRIVING, alerts, requestedLocale, legStartTime, legEndTime);
             }
 
+            if (leg.intermediateStops != null) {
+                for (StopArrival visit : leg.intermediateStops) {
+                    Place place = visit.place;
+                    if (place.stopId != null) {
+                        Collection<TransitAlert> alerts = getAlertsForStopAndRoute(graph, place.stopId, routeId);
+                        Date stopArrival = visit.arrival.getTime();
+                        Date stopDepature = visit.departure.getTime();
+                        addAlertPatchesToLeg(leg, StopCondition.PASSING, alerts, requestedLocale, stopArrival, stopDepature);
```

Comment:
@lassetyr On stand-up today you talked about a condition an alert to be applied to a intermediate stop - is there some kind of flagg for an alert to say that it apply to INTERMEDIATE_STOPS/BOARD_ALIGHT_STOPS?

Reply:
This is defined in the attribute [TransitAlert.stopConditions](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/main/java/org/opentripplanner/routing/alertpatch/TransitAlert.java#L46), with possible values from [StopCondition](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/main/java/org/opentripplanner/routing/alertpatch/StopCondition.java)